tests/test_runner/runner.py

```diff
@@ -16,5 +16,30 @@ def add_arguments(cls, parser):
         parser.add_argument('--option_b', '-b', default='2'),
         parser.add_argument('--option_c', '-c', default='3'),
 
-    def run_tests(self, test_labels, extra_tests=None, **kwargs):
+    def run_tests(self, test_labels, **kwargs):
         print("%s:%s:%s" % (self.option_a, self.option_b, self.option_c))
+
+
+class NoOpTestRunner(DiscoverRunner):
```

Comment:
Since this is only being used in one test case class, I would put it right before that test case class. If it turns out to be useful for other tests, it could always be moved to a more central location and modified as needed, etc.